packages/e2e-test-utils/src/text-content-areas-have-focus.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Internal dependencies
+ */
+import { textContentAreas } from './text-content-areas';
+
+/**
+ * Tabs through the text content areas of a block and asserts the expected values
+ *
+ * @param {string} content The expected value of the block's contenteditable elements
+ */
+
+export async function textContentAreasHaveFocus( content ) {
+	const blocks = await textContentAreas( { empty: false } );
+	const isFocusedTextContentArea = await page.evaluate( () => document.activeElement.contentEditable );
+	const textContentAreaContent = await page.evaluate( () => document.activeElement.innerHTML );
```

Comment:
`isFocusedTextContentArea` and `textContentAreaContent` need to be moved inside the loop, under the `if` block, otherwise their value won't be updated as you cycle through the content areas.

Reply:
(Doing so will cause the tests to fail, which confirms my earlier observation that the citation in the Quote block isn't being populated.)

Reply:
> It should be entering "Quote Block Content" into the main content area, as well as the lighter grey, smaller font size citation content area.

That's what I was expecting it to do, but when I ran the tests on this branch locally the citation wasn't populated. And after making the change to the loop I mentioned above, the tests failed, which is consistent with the citation field being empty.